apps/demo/src/app/modals/add-to-cart-notification/components/add-to-cart-notification/add-to-cart-notification.component.spec.ts

```diff
@@ -8,8 +8,8 @@ import { DaffProductFactory } from '@daffodil/core/testing';
 import { Product } from '@daffodil/core';
 import { DaffLoadingIconModule } from '@daffodil/design';
 
-// importing from @daffodil/state doesn't work.
-import * as fromProduct from 'libs/state/src/product/reducers/index';
+// importing from @daffodil/product doesn't work.
+import * as fromProduct from 'libs/product/src/reducers/index';
```

Comment:
Why doesn't this work, I would anticipate it should? Could you provide specifics into what's going on?

Reply:
I looked into this again, and it turns out that importing it from @daffodil/product does work, but when I try to spy on the selector for getting a product later I get the error:

`selectProduct is not declared writable or has no setter`

I don't get that error if I import from src, but I still can't get the test to work because the mocked selector always comes back with a null Product for some reason.

Reply:
I've moved this into a separate issue #285